docs/organizations/integrations/slack-integration.md

```diff
@@ -0,0 +1,35 @@
+# Organization Slack integration for Security issues
+
+The organization Slack integration for Security issues sends notifications to a Slack channel of your choice whenever Codacy detects new critical Security issues in the default branch of any repository in your organization.
+
+## Installing the Slack integration
+
+To install the Slack integration:
+
+1.  **On Slack**:
+
+    1.  [Create an Incoming Webhook](https://api.slack.com/messaging/webhooks) and set it to post to a channel of your choice.
+
+    1.  From the page [Your Apps](https://api.slack.com/apps), click the application linked to the webhook you've just created and open the associated **Incoming Webhooks** page.
+
+    1.  Copy the Incoming Webhook URL (it starts with `https://hooks.slack.com/services/`).
+
+1.  **On Codacy**:
```

Comment:
Also, since we provide all these links, we can just point users to the Integration page directly and explain the step by step from there to make it simpler, perhaps

Reply:
Yes, that's a dynaimc link 😕

[We often use](https://github.com/search?q=repo%3Acodacy%2Fdocs+%22your+organization+**%22&type=code) this pattern in the docs.